source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -531,17 +531,27 @@ def findOverlayClasses(self,clsList):
 			if clsList[0]==IAccessible and len(clsList)==3 and self.IAccessibleRole==oleacc.ROLE_SYSTEM_CLIENT and self.childCount==0:
 				clsList.insert(0,ContentGenericClient)
 
-	def __init__(self,windowHandle=None,IAccessibleObject=None,IAccessibleChildID=None,event_windowHandle=None,event_objectID=None,event_childID=None):
+	def __init__(self,windowHandle=None,IAccessibleObject=None,IAccessibleChildID=None,event_windowHandle=None,event_objectID=None,event_childID=None,parent=None,indexInParent=None):
+		"""
+		@param windowHandle: the window handle, if known.

```

Comment:
This one and a few below it don't have a cap after the colon, while other params do.